scripts/gen_jvt.py

```diff
@@ -173,18 +186,27 @@ def generate(self, download, jobs):
                         "PPH444P6_Mitsubishi_A": OutputFormat.NONE,
                         "PPH444P7_Mitsubishi_A": OutputFormat.NONE,
                         "PPH444P8_Mitsubishi_A": OutputFormat.NONE,
-                        "PPH444P9_Mitsubishi_A": OutputFormat.NONE
+                        "PPH444P9_Mitsubishi_A": OutputFormat.NONE,
```

Comment:
Why the last "," ?

Reply:
It's a matter of automatic syntax correction that black formatter does.